docs/_snippets/tags-in-meta-and-story.md

```diff
@@ -29,38 +119,44 @@ export const ExperimentalFeatureStory: Story = {
 };
 ```
 
-```svelte filename="Button.stories.svelte" renderer="svelte" language="js" tabTitle="Svelte CSF"
-<script module>
-  import { defineMeta } from '@storybook/addon-svelte-csf';
+```ts filename="Button.stories.js|ts" renderer="react" language="ts" tabTitle="CSF 4 (experimental)"
+import config from '#.storybook/preview.ts';
 
-  import Button from './Button.svelte';
+import { Button } from './Button';
 
-  const { Story } = defineMeta({
-    component: Button,
-    /*
-     * All stories in this file will have these tags applied:
-     * - autodocs
-     * - dev (implicit default, inherited from preview)
-     * - test (implicit default, inherited from preview)
-     */
-    tags: ['autodocs'],
-  });
-</script>
+const meta = config.meta({
+  component: Button,
+  /*
+   * All stories in this file will have these tags applied:
+   * - autodocs
+   * - dev (implicit default, inherited from preview)
+   * - test (implicit default, inherited from preview)
+   */
+  tags: ['autodocs'],
+});
 
-<!--
-  This particular story will have these tags applied:
-  - experimental
-  - autodocs (inherited from meta)
-  - dev (inherited from meta)
-  - test (inherited from meta)
--->
-<Story name="ExperimentalFeatureStory" tags={['experimental']} />
+export default meta;
+
+export const ExperimentalFeatureStory = meta.story({
+  /*
+   * This particular story will have these tags applied:
+   * - experimental
+   * - autodocs (inherited from meta)
+   * - dev (inherited from meta)
+   * - test (inherited from meta)
+   */
+  tags: ['experimental'],
+});
 ```
 
-```js filename="Button.stories.js" renderer="svelte" language="js" tabTitle="CSF"
-import Button from './Button.svelte';
+<!-- js & ts-4-9 (when applicable) still needed while providing both CSF 3 & 4 -->
```

Comment:
For the time being, we will need `react` versions of whatever languages are available for other renderers. In this case that means `js`, `ts`, and `ts-4-9`.

I left this comment in there for an easy find/replace once we only have CSF 4 snippets.

Reply:
If you think this is too much of a burden, I can try to adjust the snippets logic to only require one language for the CSF 4 snippets. But I anticipate that being quite tricky, and I'd rather do the "dumb" solution.